docker-compose.yml

```
@@ -334,19 +334,19 @@ services:
     command: run worker
   events-consumer:
     <<: *sentry_defaults
-    command: run ingest-consumer --consumer-type=events
+    command: run consumer ingest-events --consumer-group ingest-consumer
   attachments-consumer:
     <<: *sentry_defaults
-    command: run ingest-consumer --consumer-type=attachments
+    command: run consumer ingest-attachments --consumer-group ingest-consumer
   transactions-consumer:
     <<: *sentry_defaults
-    command: run ingest-consumer --consumer-type=transactions
+    command: run consumer ingest-transactions --consumer-group ingest-consumer
   ingest-replay-recordings:
     <<: *sentry_defaults
-    command: run ingest-replay-recordings
+    command: run consumer ingest-replay-recordings --consumer-group ingest-replay-recordings
   ingest-profiles:
     <<: *sentry_defaults
-    command: run ingest-profiles --no-strict-offset-reset
+    command: run consumer ingest-profiles --consumer-group ingest-profiles --no-strict-offset-reset
```

I think this change may fix the issue:
```suggestion
    command: run consumer --no-strict-offset-reset ingest-profiles --consumer-group ingest-profiles
```

Reason being, the new `consumer` command expecting a positional `topic` argument at the end and I suspect putting `--no-strict-offset-reset` at the end might be confusing that.

---

thanks for the suggestion, let me try that